lib/generators/rolemodel/mailers/mailers_generator.rb

```diff
@@ -0,0 +1,62 @@
+require_relative '../../bundler_helpers'
+
+
+module Rolemodel
+  class MailersGenerator < Rails::Generators::Base
+    include Rolemodel::BundlerHelpers
+    source_root File.expand_path('templates', __dir__)
+
+    def install_premailer_rails
+      gem 'premailer-rails'
+      run_bundle
+    end
+
+    def add_premailer_rails_config
+      copy_file 'config/initializers/premailer_rails.rb'
+    end
+
+    def add_action_mailer_asset_host
+      unless File.exist?(Rails.root.join('config/initializers/devise.rb'))
+        inject_into_file 'config/environments/development.rb', after: "config.action_mailer.perform_caching = false\n" do
+          optimize_indentation <<~'RUBY', 2
+
+            # Default mailing host suggested by Devise installation instructions
+            config.action_mailer.default_url_options = { host: 'localhost', port: 3000 }
+          RUBY
+        end
+      end
+      
+      inject_into_file 'config/environments/development.rb', after: "config.action_mailer.default_url_options = { host: 'localhost', port: 3000 }\n" do
+        optimize_indentation <<~'RUBY', 2
+
+          # Ensure premailer_rails can point to webpack compiled resources
+          # https://github.com/fphilipe/premailer-rails/issues/232#issuecomment-839819705
+          config.action_mailer.asset_host = 'http://localhost:3000'
```

Comment:
Is this only needed for Development?

Reply:
Good question. I think yes since styles are compiled via webpacker and premailer_rails doesn't know about them by default. In production, your styles are precompiled and would not be served up by the webpack dev server so we shouldn't have issues.